package.json

```diff
@@ -417,32 +437,6 @@
                     "group": "Julia@30"
                 }
             ],
-            "commandPalette": [
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-delete-all"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-delete"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-next"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-first"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-last"
-                },
-                {
-                    "when": "false",
-                    "command": "language-julia.plotpane-previous"
-                }
-            ],
```

Comment:
Obviously not part of this PR, but what happened to all of these commands? Or are they still here and this is just removing a condition?

Reply:
Yeah, this just unhides them in the command pane.